src/plugins/intel_cpu/src/nodes/mkldnn_lrn_node.cpp

```diff
@@ -8,10 +8,54 @@
 #include <ngraph/opsets/opset1.hpp>
 #include <memory_desc/cpu_memory_desc_utils.h>
 #include "memory_desc/dnnl_blocked_memory_desc.h"
+#include <common/primitive_hashing_utils.hpp>
 
 using namespace MKLDNNPlugin;
 using namespace InferenceEngine;
 
+namespace {
+struct LrnKey {
+    DnnlMemoryDescCPtr inp0;
+    impl_desc_type implType;
+    mkldnn::algorithm alg;
+    size_t size;
+    int k;
+    float alpha;
+    float beta;
+
+    size_t hash() const;
+    bool operator==(const LrnKey& rhs) const;
+};
+
+size_t LrnKey::hash() const {
+    using namespace dnnl::impl;
+    using namespace dnnl::impl::primitive_hashing;
+
+    size_t seed = 0;
+
+    seed = hash_combine(seed, get_md_hash(inp0->getDnnlDesc().data));
+    seed = hash_combine(seed, implType);
+    seed = hash_combine(seed, alg);
+    seed = hash_combine(seed, size);
+    seed = hash_combine(seed, k);
+    seed = hash_combine(seed, alpha);
+    seed = hash_combine(seed, beta);
+
+    return seed;
+}
+
+bool LrnKey::operator==(const LrnKey &rhs) const {
+    bool retVal = true;
+    if (inp0 != rhs.inp0) {
+        retVal = retVal && inp0 && rhs.inp0 && inp0->getDnnlDesc() == rhs.inp0->getDnnlDesc();
+    }
+
+    retVal = retVal && implType == rhs.implType && alg == rhs.alg && alg == rhs.alg && size == rhs.size && k == rhs.k &&
```

Comment:
```suggestion
    retVal = retVal && implType == rhs.implType && alg == rhs.alg && size == rhs.size && k == rhs.k &&
```

Reply:
@mangguo321 please add the fix in https://github.com/openvinotoolkit/openvino/pull/9549

Reply:
OK. Thanks @dmitry-gorokhov